app/ui/src/app/app.module.ts

```diff
@@ -20,8 +20,8 @@ import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app.routing';
 import { SyndesisCommonModule } from './common';
 import { appConfigInitializer, ConfigService } from './config.service';
-import { StoreModule as LegacyStoreModule } from '@syndesis/ui/store/store.module';
-import { platformReducer, SYNDESIS_GUARDS } from './platform';
+import { StoreModule as LegacyStoreModule } from './store/store.module';
+import { platformReducer, PlatformEffects, platformEndpoints, SYNDESIS_GUARDS } from './platform';
```

Comment:
Same question regarding utilizing "@syndesis/ui" path alias

Reply:
As you can see I removed those on purpose. Let’s stick to relative paths when consuming tokens from root classes.